packages/admin/package.json

```diff
@@ -39,11 +37,17 @@
         "react": "^16.8",
         "react-dom": "^16.8",
         "react-final-form": "^6.3.1",
+        "react-dnd": "^14.0.2",
         "react-intl": "^5.10.0",
         "react-router": "^5.1.2",
         "react-router-dom": "^5.1.2",
         "styled-components": "^4.0.0 || ^5.0.0"
     },
+    "peerDependenciesMeta": {
+        "react-dnd": {
```

Comment:
React dnd must be a peer-dependency  now and no longer a dependency - because the application must be wrapped inside a DndProvider.

I think the peer-dependency can be kept optional (peerDependenciesMeta is supported since npm 7), because as long the application doesn't use `TableDndOrder` the DndProvider is not needed.